ufficio_soci/tests.py

```diff
@@ -2263,4 +2266,64 @@ def test_cancellazione_quota_socio(self):
                             causale="Quota", importo=8.0)
 
         sessione_delegato_territoriale.visit("%s/us/ricevute/%d/annulla/" % (self.live_server_url, quota.pk))
-        self.assertTrue(sessione_delegato_territoriale.is_text_present("ANNULLATA"))
\ No newline at end of file
+        self.assertTrue(sessione_delegato_territoriale.is_text_present("ANNULLATA"))
+
+
+    @freeze_time('2018-03-21')
+    def test_download_iscritti(self):
+
+        # crea oggetti e appartenza
+        presidente = crea_persona()
+        direttore_corso = crea_persona()
+        aspirante1 = crea_persona()
+        aspirante2 = crea_persona()
+        sostenitore1 = crea_persona()
+        sostenitore2 = crea_persona()
+        sede = crea_sede(presidente)
+        crea_appartenenza(direttore_corso, sede, tipo=Appartenenza.DIPENDENTE)
+        crea_appartenenza(sostenitore1, sede, tipo=Appartenenza.SOSTENITORE)
+        crea_appartenenza(sostenitore2, sede, tipo=Appartenenza.SOSTENITORE)
+
+        # crea locazione, aspiranti e corso
+        locazione = crea_locazione()
+        Aspirante.objects.create(raggio=900, locazione=locazione, persona=aspirante1)
+        Aspirante.objects.create(raggio=900, locazione=locazione, persona=aspirante2)
+        corso_base = CorsoBase.objects.create(locazione=locazione, stato=Delega.ATTIVA, sede=sede,
+                                              data_inizio=poco_fa(),
+                                              data_esame=datetime.date(2018, 10, 10),
+                                              anno=2018, progressivo=1)
+
+        # nomina delegato corso
+        ctype = ContentType.objects.get_for_model(CorsoBase)
+        Delega.objects.create(persona=direttore_corso, stato=Delega.ATTIVA, tipo=DIRETTORE_CORSO,
+                              inizio=poco_fa(), oggetto_tipo=ctype,
```

Comment:
Non credo sia necessario `ctype`, credo che ti basti passare `oggetto=corso_base` in questa istanza. Django dovrebbe popolare il campo `oggetto_tipo` automaticamente.